deploy/release.yaml

```diff
@@ -4,11 +4,15 @@ kind: ServiceAccount
 metadata:
   name: knative-openshift-ingress
   namespace: knative-serving
+  labels:
+    serving.knative.dev/release: v0.0.6
```

Comment:
The `v0.0.6` should be `devel` in the git repo? Then, we replace the `devel` with `v0.0.6` for the release yaml. (It is the same manner with upstream's - e.g https://github.com/knative/serving/blob/master/config/100-namespace.yaml#L21)

Reply:
Maybe. Though we somewhat hardcode the version in the repository for now anyway 🤷‍♂